Database/QuestieDB.lua

```diff
@@ -1,4 +1,34 @@
 ---@class QuestieDB
+---@field npcCompilerTypes table
+---@field npcCompilerOrder table
+---@field npcKeys table
+---@field npcKeysReversed table
+---@field questCompilerTypes table
+---@field questCompilerOrder table
+---@field questKeys table
+---@field questKeysReversed table
+---@field objectCompilerTypes table
+---@field objectCompilerOrder table
+---@field objectKeys table
+---@field objectKeysReversed table
+---@field itemCompilerTypes table
+---@field itemCompilerOrder table
+---@field itemKeys table
+---@field itemKeysReversed table
+---@field npcData any
+---@field questData any
+---@field objectData any
+---@field itemData any
```

Comment:
Those should also be `table` I think

Reply:
Done in https://github.com/Questie/Questie/pull/3115/commits/d7d473bcf3757d69408e589e2bbe706d07fb4517